extension/src/ohlc.rs

```diff
@@ -14,65 +15,164 @@ use tspoint::TSPoint;
 pub mod toolkit_experimental {
     use super::*;
 
+    flat_serialize_macro::flat_serialize! {
+        #[derive(Serialize, Deserialize, Debug, Copy)]
+        enum VolKind {
+            storage_kind: u64,
```

Comment:
I was going to ask what's up with this unused and strangely named value, but I figure I better test it out first :)

I see that it's clearly, surprisingly, quietly, and undocumentedly required, and required first!.

Looks like the name doesn't matter though.  How about `unused_but_required_by_flat_serialize` or something?

Reply:
I copied `storage_kind` from [here](https://cs.github.com/timescale/timescaledb-toolkit/blob/6791f1ba377342f8e557c89f9ac955590f22457d/extension/src/hyperloglog.rs#L178):
```rust
flat_serialize_macro::flat_serialize! {
    #[derive(Debug, Serialize, Deserialize)]
    enum Storage<'a> {
        storage_kind: u64,
```
so it's possible the convention is `$(enum_name)_kind` but I like your suggestion :)